src/services/faultTreeService.tsx

```diff
@@ -186,10 +185,9 @@ export const getTreePathsAggregate = async (): Promise<[FaultEvent[]]> => {
 export const calculateCutSets = async (faultTreeUri: string) => {
   try {
     const fragment = extractFragment(faultTreeUri);
-    const response = await axiosClient.put(`/faultTrees/${fragment}/cutsets`, null, {
+    const response = await axiosClient.put(`/faultTrees/${fragment}/evaluate`, null, {
```

Comment:
I do not know if /evaluate or /cutsets is correct to call here .. just pointing out, that you changed it here :) Please, confirm it is ok.

Reply:
It's correct. Probably I already changed it in other PR.